crafters/image/ImageReader/README.md

```diff
@@ -1,3 +1,49 @@
 # ImageReader
 
-loads the image from the given file path and save the `ndarray` of the image in the Document. 
\ No newline at end of file
+**ImageReader** is a crafter that loads an image from a given file path and save the `ndarray` of the image in the Document. It reads the image specified in `buffer` and save the `ndarray` of the image in the `blob` of the document.
```

Comment:
As far as I remember it also can work from bytes right?

Reply:
Yeah, I changed it already